parsimonious/tests/test_grammar.py

```diff
@@ -378,6 +394,26 @@ def test_unconnected_custom_rules(self):
         eq_(grammar.parse(s),
             Node(grammar['one_char'], s, 0, 1))
 
+    def test_callability_of_routines(self):
+        ok_(is_callable(function_rule))
+        ok_(is_callable(self.method_rule))
+        ok_(is_callable(self.rules["descriptor_rule"]))
```

Comment:
Let's keep using single quotes where we can, to keep consistent with the rest of the codebase. I should have the linter flag that if it doesn't already.